Tests/Marketplace/zip_packs.py

```diff
@@ -114,61 +114,110 @@ def remove_test_playbooks_from_signatures(path, filenames):
+    packs_list = [pack.name for pack in os.scandir(PACKS_FULL_PATH)]  # list of all packs from repo
+
+    for entry in dir_entries:
+        entry_path = os.path.join(zip_path, entry)
+        if entry not in IGNORED_FILES and entry in packs_list and os.path.isdir(entry_path):
+            # This is a pack directory, should keep only most recent release zip
+            pack_files = get_pack_files(entry_path)
+            latest_zip = get_latest_pack_zip_from_blob(entry, pack_files)
+            if not latest_zip:
+                logging.warning(f'Failed to get the zip of the pack {entry} from GCP')
+                continue
+            print(f"Found latest zip of {entry}, which is {latest_zip}")
+            zipped_packs.append({Path(latest_zip).stem: latest_zip})
 
     if not zipped_packs:
         logging.critical('Did not find any pack to download from GCP.')
         sys.exit(1)
     return zipped_packs
 
 
+def download_packs_from_gcp(storage_bucket_name, gcp_path, destination_path, circle_build, branch_name):
+    """
+    Copies all content in the gcp path to destination_path
+    Args:
+        storage_bucket_name: The name of the GCP bucket to download from.
+        gcp_path: The path of the packs in the GCP bucket.
+        destination_path: The path to download the packs to.
+        branch_name: The branch name of the build.
+        circle_build: The number of the circle ci build.
+    """
+
+    if gcp_path == BUILD_GCP_PATH:
+        src_path = os.path.join('gs://', storage_bucket_name, gcp_path, branch_name, circle_build, 'content', 'packs')
+    else:
+        src_path = os.path.join('gs://', storage_bucket_name, gcp_path, branch_name, circle_build)
+
+    if not branch_name or not circle_build:
+        src_path = src_path.replace('/builds/content', '')
+
+    try:
+        process = subprocess.Popen(["Tests/scripts/cp_gcp_dir.sh", src_path, destination_path])
+        out, err = process.communicate()
+        if out:
+            print("out" + out.decode('utf-8'))
+        if err:
+            print("err" + err.decode('utf-8'))
+    except Exception as e:
+        logging.critical(f"Failed to run cp_gcp_dir.sh, Error:{e}")
+        sys.exit(1)
+
+
+def copy_to_other_dir(zipped_packs):
+    """
+    Copies zip files if needed
+    Args:
+        zipped_packs: A list of dictionaries containing pack name as key and it's latest zip path as value
+    """
+    if os.path.exists('/home/runner/work/content-private/content-private/content/artifacts/'):
+        for zip_pack in zipped_packs:
+            for name, path in zip_pack.items():
+                logging.info(f"Copying pack from {path} to /home/runner/work/content-private/"
+                             f"content-private/content/artifacts/packs/{name}.zip")
+                shutil.copy(path,
+                            f'/home/runner/work/content-private/content-private/content/artifacts/'
+                            f'packs/{name}.zip')
+
```

Comment:
Please handle `Shutil.Error`

Reply:
Sure, added handling.

Reply:
> I don't see that you addressed @guyfreund's comment.

You can see at line 270 that I added a try except surrounding this function call to handle it's errors. Due to decision made by me and @teizenman to handle all excetpions in main.